prisma/migrations/20210529200211_add_tables_for_nextjs_auth/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
```

Comment:
What's the reason we are doing a migration for this rather than have a `schema.sql` file in the `prisma` directory?

Reply:
https://github.com/uwblueprint/richmond-centre-for-disability/tree/feature/initialize-tables/prisma Here is how it's done on RCD currently. Here is dancefest: https://github.com/uwblueprint/dancefest-web/tree/staging/prisma